test/contracts/AuctionRegistrar.cpp

```diff
@@ -67,7 +67,7 @@ contract AuctionSystem {
 	function onAuctionEnd(string _name) internal;
 
 	function bid(string _name, address _bidder, uint _value) internal {
-		var auction = m_auctions[_name];
+		Auction auction = m_auctions[_name];
```

Comment:
Please add `storage` or `memory` while you are at it to all these, i.e. this should be `Action storage auction = m_auctions[_name]`.

Reply:
What was the `var` keyword doing? defaulting to `storage`, too?

Reply:
var uses whatever is on the right hand side. If you specify the type, then it defaults to storage.